app/routes/cdo/create/dog-details.js

```diff
@@ -71,12 +72,18 @@ const validatePayload = (payload) => {
 module.exports = [
   {
     method: 'GET',
-    path: routes.details.get,
+    path: `${routes.details.get}/{dogId?}`,
     options: {
       auth: { scope: [admin] },
       handler: async (request, h) => {
         const dog = getDog(request)
 
+        if (dog === undefined) {
+          return h.response().code(404).takeover()
+        }
+
+        dog.id = request.params.dogId
+
         const { breeds } = await getBreeds()
```

Comment:
Do you have any thoughts on caching dog breeds, or police forces etc rather than us having ro re-read them from the API each time? Or does the API currently handle its own caching?